mappings/net/minecraft/entity/ai/goal/WolfBegGoal.mapping

```diff
@@ -2,7 +2,10 @@ CLASS alu net/minecraft/entity/ai/goal/WolfBegGoal
 	FIELD a owner Larg;
 	FIELD b begFrom Lavm;
 	FIELD c world Lbgv;
-	FIELD d chance F
+	FIELD d begDistance F
 	FIELD e timer I
+	FIELD f validPlayerPredicate Lapp;
 	METHOD <init> (Larg;F)V
 		ARG 1 owner
+		ARG 2 begDistance
+	METHOD a playerHoldingValidItem (Lavm;)Z
```

Comment:
`isPlayerHoldingValidItem` or `isAttractive` (I prefer `isAttractive` as it conveys the meaning while it is short)